spec/controllers/api/v1/classifications_controller_spec.rb

```diff
@@ -228,7 +228,7 @@ def create_gold_standard
         end
 
         it 'should be filterable by a list of subject ids' do
-          get :gold_standard, subject_ids: gs.subject_ids.join(',')
```

Comment:
I though the same thing and then wondered if anyone was using it as it's not documented. Perhaps i'll just add a before filter in the controller to map it to the right key for that filter.